io/zenoh-transport/src/unicast/manager.rs

```diff
@@ -565,19 +565,13 @@ impl TransportManager {
                 peer_id,
             };
 
-            let res = super::establishment::accept::accept_link(&link, &c_manager, &mut auth_link)
-                .timeout(c_manager.config.unicast.accept_timeout)
-                .await;
-            match res {
-                Ok(res) => {
-                    if let Err(e) = res {
-                        log::debug!("{}", e);
-                    }
-                }
-                Err(e) => {
-                    log::debug!("{}", e);
-                    let _ = link.close().await;
-                }
+            if let Err(e) =
```

Comment:
This check shouldn't be changed. `timeout()` returns a `Result`, if `Ok` it contains the `Result` of `accept_link`. In practice it's a `Result<Result<_, _>, _>`. So, the proposed change only catches the `Error` generated by the `timeout`.

Reply:
Yes but nothing was done except logging in case `Ok(Err(_))`. So as we don't want to log this case any more (logging occurs in `accept_link`) there is no point checking this case any more.